### DISCUSSION TITLE
New release with commit 6afe5768381599dfbbe53b69d352f618d8a45834

### DISCUSSION THREAD

**Discussion by tpaimparay** [keycloak/keycloak#13059] [Q&A]
Hello,
I am a little bit lost because release 18.0.1 and 18.0.2 seems not embeded commit 6afe5768381599dfbbe53b69d352f618d8a45834 that "Make ImagePullPolicy configurable".
This commit is present in the main branch since 2022-04-29  but not in tag 18.0.1 and 18.0.2 that were set later.

Could you help me to know if this feature is present or not in last releases ?

if not, do you know when is scheduled the next release ?
Thanks

**Comment by abstractj** [comment]
@tpaimparay you are correct, I could not find the commit in the keycloak-operator for `18.0.2`. @vmuzikar do you know why https://github.com/keycloak/keycloak-operator/pull/506 is not part of 18.0.2 release?

**Reply by vmuzikar** [reply]
@tpaimparay Hello, the affected commit is not yet part of any release as it was merged after 18.0.0 was released. The 18.0.1 and 18.0.2 releases were only microreleases that usually include only critical fixes like CVEs, that's the reason it's not part of them either. New features are added usually in major releases. The next one is 19.0.0 (which is also the last major release for this legacy Operator). Unfortunately, I don't have any specific ETA to share with you but AFAIK it should be relatively soon, no more than a few weeks.
